jenkins/osa_onmetal.functions.groovy

```diff
@@ -260,10 +262,10 @@ def install_persistent_resources_tests() {
     String host_ip = get_onmetal_ip()
 
     // Install Persistent Resources tests on the onMetal host
-    echo 'Installing Persisten Resources Tempest Plugin on the onMetal host'
+    echo 'Installing Persistent Resources Tempest Plugin on the onMetal host'
     sh """
     ssh -o StrictHostKeyChecking=no root@${host_ip} '''
-    git clone https://github.com/osic/persistent-resources-tests.git /root/persistent-resources-tests
+    git clone https://github.com/antwash/persistent-resources-tests.git /root/persistent-resources-tests
```

Comment:
Pointing to Ant's git repo.
Need to revert after pending changes are committed on osic one